### PR TITLE
add `project` option to build from subdirectory

### PR DESCRIPTION
**Pull request by simeonschaub**
addresses #26

### CODE REVIEW

**Comment by tpoisot** [comment]
Not to necrobump but -- this would be really good to have (especially to simplify deploying the docs for monorepos)

**Comment by laggvar** [comment]
Joining the crowd asking to add this feature. Subfolders are supported by the `project` flag for `julia-runtest` so it would make sense to also support subfolders for documentation.

File: action.yml [hunk 1]
```diff
@@ -36,8 +42,10 @@ runs:
         # Run the Julia command
         "${julia_cmd[@]}"
       shell: bash
-      if: ${{ inputs.install-package == 'true'}}
-    - run: |
+      working-directory: ${{inputs.project}}
+      if: ${{ inputs.install-package }}
```
Comment (IanButterworth) [inline action.yml:46]:
```suggestion
      if: ${{ inputs.install-package == 'true'}}
```

**Review by IanButterworth — APPROVED** [review]
LGTM

**Pull request merged**